Sources/OpenAPIRuntime/Conversion/Converter+Common.swift

```diff
@@ -17,31 +17,68 @@ extension Converter {
 
     // MARK: Miscs
 
-    /// Validates that the Content-Type header field (if present)
```

Comment:
One method was removed/moved to Deprecated, and new ones were added. Easier to review in split mode.